src/components/App.jsx

```diff
@@ -6,9 +6,11 @@ import flag from 'cozy-flags'
 const App = () => {
   const embeddedChatUrl = flag('chat.embedded-app-url')
 
-  useExternalBridge(embeddedChatUrl)
+  const isReady = useExternalBridge(embeddedChatUrl)
 
-  return <iframe id="embeddedApp" src={embeddedChatUrl}></iframe>
+  return (
```

Comment:
why not returning null if not ready ?

Reply:
I added a comment to explain it 

```
// We can not return null if bridge is not ready because to setup
// the bridge we need iframe HTML element
```